stable/jenkins/values.yaml

```diff
@@ -97,10 +97,14 @@ master:
   # Enable Kubernetes Liveness and Readiness Probes
   # ~ 2 minutes to allow Jenkins to restart when upgrading plugins. Set ReadinessTimeout to be shorter than LivenessTimeout.
   healthProbes: true
-  healthProbesLivenessTimeout: 90
-  healthProbesReadinessTimeout: 60
-  healthProbeReadinessPeriodSeconds: 10
-  healthProbeLivenessFailureThreshold: 12
+  healthProbesLivenessTimeout: 20
+  healthProbesReadinessTimeout: 20
+  healthProbeLivenessPeriodSeconds: 30
+  healthProbeReadinessPeriodSeconds: 30
+  healthProbeLivenessFailureThreshold: 5
+  healthProbeReadinessFailureThreshold: 3
+  healthProbeLivenessInitialDelay: 120
```

Comment:
`healthProbeLivenessInitialDelay` should be 90 as it has been rendered incorrectly using this variable:
`initialDelaySeconds: {{ .Values.master.healthProbesLivenessTimeout }}` and `healthProbesLivenessTimeout` was set to `90`